src/executables/master_server_monitor.cpp

```diff
@@ -477,6 +477,7 @@ server_monitor::server_entry::proc_make_arg (const std::string &args)
     {
       m_argv[i] = new char[arg.size () + 1];
       std::copy (arg.begin (), arg.end (), m_argv[i]);
+      m_argv[i][arg.size()] = '\0';
       i++;
     }
   m_argv[args.size()] = nullptr;
```

Comment:
```suggestion
  m_argv[i] = NULL;
```
m_argv는 최종적으로 execv의 인자로 넘겨집니다. exec family가 받는 배열은 NULL-terminated로, NULL을 통해 끝을 인지하기 때문에 NULL이 인자들 바로 뒤에 위치 해야 합니다.

Reply:
수정하였습니다.